docs/detections/rules-ui-create.asciidoc

```diff
@@ -263,35 +263,100 @@ For example, if a rule has an interval of 5 minutes, no additional look-back tim
+| WHERE host_count > 20
+----
+
+- This query starts by searching logs from indices that match the pattern `logs-*`. 
+- The query then aggregates the count of events by `host.name`.
+- Next, it sorts the result by `host_count` in descending order.
+- Then, it filters events where the `host_count` field is encountered more than 20 times during the specified rule interval.
+
+NOTE: Rules that use aggregating queries might create duplicate alerts. This can happen  when events that occur in the additional look-back time are aggregated twice: once in the current rule execution and once in the a previous rule execution.
+
+[float]
+[[esql-non-agg-query]]
+===== Non-aggregating query 
+A non-aggregating query doesn't use `STATS...BY` functions and doesn't aggregate source event data. Generated alerts contain source event data and fields returned by the ES|QL query. 
+
+Here is an example of a non-aggregating query:
+[source,esql]
+-----
+FROM logs* [metadata _id, _index, _version]
+| WHERE event.category == "process"  AND event.id == "8a4f500d"
+| LIMIT 10
 -----
+- This query starts by querying logs from indices that match the pattern `logs-*`. The `[metadata _id, _index, _version]` operator allows <<esql-non-agg-query-dedupe,alert deduplication>>.  
+- Next, the query filters events where the `event.category` is a process and the `event.id` is `8a4f500d`. 
+- Then, it limits the output to the top 10 results.
 
-.. **Non-aggregating query**: This is a query that _does not_ use the `STATS...BY` grouping commands. Each row in the query results can be tracked to a source document in Elasticsearch. 
-+
-For this type of query, use the operator `[metadata _id, _index, _version]` after defining the index source. This will allow for alerts to be deduplicated and linked to the source documents. For example:
-+
-[esql]
+[float]
+[[esql-non-agg-query-dedupe]]
+===== Turn on alert deduplication for rules using non-aggregating queries
+
+To deduplicate alerts, the query needs access to the `_id`, `_index`, and `_version` metadata fields within the queried source event documents. You can allow this by adding the `[metadata _id, _index, _version]` operator after the `FROM` source command, for example:
+
+[source,esql]
 -----
 FROM logs* [metadata _id, _index, _version]
-| WHERE event.id == "test"
+| WHERE event.category == "process"  AND event.id == "8a4f500d"
 | LIMIT 10
 -----
-+
-Ensure, metadata properties `id`, `_index`, `_version` are carried over through pipe operators.
 
-. Click *Continue* to <<rule-ui-basic-params, configure basic rule settings>>.
+Once the metadata fields are provided, a unique alert ID is created for each alert generated by the query.
+
+As you continue developing the query, make sure you don't {ref}/esql-commands.html#esql-drop[`DROP`] or filter out the `_id`, `_index`, or `_version` metadata fields. 
+
+Here is an example of an invalid query that uses the `DROP` command to omit the `_id` property from the results table:
```

Comment:
Are the examples invalid, or would they just fail to deduplicate alerts?

Reply:
My understanding is that they'd fail to dedupe alerts. Is this version more clear?

```suggestion
Here is an example of a query that fails to deduplicate alerts. It uses the `DROP` command to omit the `_id` property from the results table:
```

Reply:
Yes 100%. Looks great!

Reply:
Yes 100%. Looks great!